Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM python:3.7-slim-buster as ge-build
+RUN apt-get update
+# use same version of ge as wemmick
+COPY requirements-base.txt requirements-base.txt
+RUN pip wheel --wheel-dir=dist `head -n1 requirements-base.txt`
```

Comment:
not sure if this i actually a good idea but _think_  the pinned ge version needs to be part of the wheel otherwise we should in stall GE with ver as a build arg and use the same in the image tag